pkgs/games/gweled/default.nix

```diff
@@ -1,22 +1,26 @@
 { lib, stdenv, fetchbzr, intltool
 , gtk2, wrapGAppsHook, autoreconfHook, pkg-config
-, libmikmod, librsvg, libcanberra-gtk2, hicolor-icon-theme }:
+, libmikmod, librsvg, libcanberra-gtk2, hicolor-icon-theme, gettext }:
 
 stdenv.mkDerivation rec {
   pname = "gweled";
-  version = "unstable-2018-02-15";
+  version = "unstable-2021-02-11";
 
   src = fetchbzr {
     url = "lp:gweled";
-    rev = "94";
-    sha256 = "01c38y4df5a06wqbsmsn8ysxx7hav9yvw6zdwbc9m5m55z7vmdb8";
+    rev = "108";
+    sha256 = "sha256-rM4dgbYfSrVqZwi+xzKuEtmtjK3HVvqeutmni1vleLo=";
   };
 
   doCheck = false;
 
+  postPatch = ''
+    substituteInPlace configure.ac --replace "AM_GNU_GETTEXT_VERSION([0.19.8])" "AM_GNU_GETTEXT_VERSION([${gettext.version}])"
+  '';
+
   nativeBuildInputs = [ wrapGAppsHook intltool autoreconfHook pkg-config ];
 
-  buildInputs = [ gtk2 libmikmod librsvg hicolor-icon-theme libcanberra-gtk2 ];
+  buildInputs = [ gtk2 libmikmod librsvg hicolor-icon-theme libcanberra-gtk2 gettext ];
```

Comment:
gettext should go in nativeBuildInputs. Usually just needs to be ran as part of a build.

Reply:
Oops, fixed. Also removed `intltool` since `gettext` replaced it.